umh-core/pkg/fsm/connection/README.md

```diff
@@ -0,0 +1,3 @@
+Please note that the actions.go needs to be adjusted to the new error handling, specifically the ignoring of connectionservice.ErrRemovalPending. See also benthos/actions.go for the same issue.
```

Comment:
Consider renaming this to TODO.md to highlight it's significance.

Reply:
Will do